lib/auth/index.tsx

```diff
@@ -48,7 +47,6 @@ export class Auth extends Component<Props> {
 
   render() {
     // Don't render this component when running on the web
-    const config = getConfig();
     if (config.is_app_engine) {
```

Comment:
What happens here? 

With my limited Electron + Webpack + TS understanding, I think at this point `config` would be `undefined`, making the if below unreachable. Unless... `config` is somehow injected?

Reply:
It's because `config` is actually set as a Webpack plugin, defined here:

https://github.com/Automattic/simplenote-electron/blob/13cb5d1b70ac095b2d41339f6287e7ecd350a371/webpack.config.js#L157

With my limited knowledge that is why it exists everywhere :)

Reply:
Yep exactly, as long as you *don't* define it inside the file, webpack will inject it. (If it IS defined in the file then webpack will assume you want a locally-defined version instead.)

Reply:
Thanks for the info @roundhill @codebykat